docs/pages/button.js

```diff
@@ -259,245 +229,481 @@ card(
+<Flex gap={2}>
+  <Button text="Kontoeinst..." size="lg" color="gray" />
+  <Button text="Neues We..." size="lg" color="red" />
+</Flex>
+`}
+      />
+      <MainSection.Card
+        cardSize="md"
+        type="do"
+        description="Keep the Button text as simple and actionable as possible. Refer to the [Button writing guidelines](#Writing) for more detail."
+        defaultCode={`
+  <Button text="Create account" size="lg" color="red" />
+`}
+      />
+      <MainSection.Card
+        cardSize="md"
+        type="don't"
+        description="Do not add icons to a Button to refinforce the text."
+        defaultCode={`
+          <Button text="Create new Pinterest account" size="lg" color="red" iconEnd='person-add' />
+`}
+      />
+    </MainSection.Subsection>
+  </MainSection>,
 );
 
 card(
-  <Example
-    name="Size"
-    id="size"
-    defaultCode={`<Flex gap={2}>
-  <Button size="sm" text="Small-sized button" />
-  <Button text="Medium-sized button" />
-  <Button size="lg" text="Large-sized button" />
-</Flex>`}
-  />,
+  <MainSection name="Accessibility">
+    <MainSection.Subsection
+      title="ARIA attributes"
+      description={`
+When Button text does not provide sufficient context about the Button’s behavior, supply a short, descriptive label for screen-readers using accessibilityLabel. Texts like “Click here“, “Follow“, or “Shop“ can be confusing when a screen reader reads them out of context. In those cases, we must pass an alternative text with deeper context to replace the Button text, like “Follow Ryan” or “Shop Wedding Invitations”.
+If Button is used as a control Button to show/hide a Popover-based component, we recommend passing the following ARIA attributes to assist screen readers:
+- \`accessibilityControls\`: informs the screen reader that Button controls the display of an anchored Popover-based component. It populates [aria-controls](https://www.w3.org/TR/wai-aria-practices/examples/menu-button/menu-button-links.html).
+- \`accessibilityHaspopup\`: informs the screen reader that there’s a Popover-based component attached to Button. It populates [aria-haspopup](https://www.w3.org/TR/wai-aria-practices/examples/menu-button/menu-button-links.html).
+- \`accessibilityExpanded\`: informs the screen reader whether the button-anchored Popover-based component is currently open or closed. It populates [aria-expanded](https://www.w3.org/TR/wai-aria-practices/examples/menu-button/menu-button-links.html).
+`}
+    />
+  </MainSection>,
 );
 
 card(
-  <Combination
-    id="color"
-    name="Color"
-    color={['gray', 'red', 'blue', 'white', 'transparent', 'semiTransparentWhite']}
-  >
-    {(props, i) => <Button id={`example-${i}`} onChange={() => {}} {...props} text="Button" />}
-  </Combination>,
+  <MainSection
+    name="Localization"
+    description="Be sure to localize `text` and `accessibilityLabel`. Note that localization can lengthen text by 20 to 30 percent. Avoid truncating Button text whenever possible."
```

Comment:
I'd refer back to best practices for truncation

Reply:
Done!